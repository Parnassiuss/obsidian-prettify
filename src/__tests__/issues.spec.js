// @ts-nocheck
const Remarkable = require("remarkable");
const prettify = require("../index");

function parseContents(
    content,
    settings = {}
) {
    return new Remarkable.Remarkable()
        .use(prettify)
        .render(content);
}

describe("basic", () => {

    test("https://github.com/cristianvasquez/obsidian-prettify/issues/1", () => {

        const mock = `# issues/1
Prettifier:
- [ ] Don't break my TODO`
        expect(parseContents(mock)).toMatchSnapshot();
    });

    test("https://github.com/cristianvasquez/obsidian-prettify/issues/1", () => {

        const mock = `# issues/1

1. fjlkefjl
    1. feafe
2. feafea
3. feaf
    1. fea
3. fefe`

        expect(parseContents(mock)).toMatchSnapshot();
    });

});
