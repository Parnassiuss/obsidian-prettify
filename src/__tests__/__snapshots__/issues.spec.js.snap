// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic https://github.com/cristianvasquez/obsidian-prettify/issues/1 1`] = `
"# issues/1

Prettifier:

- [ ] Don't break my TODO
"
`;

exports[`basic https://github.com/cristianvasquez/obsidian-prettify/issues/1 2`] = `
"# issues/1

1. fjlkefjl
  2. feafe
2. feafea
3. feaf
  1. fea
4. fefe
"
`;
